src/sensor/ping360.cpp

```diff
@@ -40,9 +40,20 @@ Ping360::Ping360()
 
     connect(&_timeoutProfileMessage, &QTimer::timeout, this, [this] {
         qCWarning(PING_PROTOCOL_PING360) << "Profile message timeout, new request will be done.";
+
         // Since ping360 uses automatic baudrate detection
         // it's necessary to start the connection to force baud rate changes
+        if(link()->type() == LinkType::Serial)
+        {
+            SerialLink* serialLink = dynamic_cast<SerialLink*>(link());
+            if(serialLink) {
+                serialLink->port()->setBaudRate(_firmwareMaxBaudRate);
```

Comment:
We cannot force the baudrate. There are situations where communications will succeed @115200 (protocol detector), then fail @2000000 here. High baudrates are not sustainable with longer tether cable lengths.